### PR TITLE
Meteor rn reachability url pr

### PR DESCRIPTION
**Pull request by bratelefant**
Tested on iOS and android, on iOS I also checked via a proxy if, when using `reachabilityUrl` option, googles servers are no longer contacted.

### CODE REVIEW

File: src/Meteor.js [hunk 1]
```diff
@@ -113,7 +113,15 @@ const Meteor = {
 
     try {
       const NetInfo = require('@react-native-community/netinfo').default;
+      
+      if (options.reachabilityUrl)
```
Comment (jankapunkt) [inline src/Meteor.js:117]: Please add brackets here.

**Review by jankapunkt — CHANGES_REQUESTED** [review]
Please run `npm run lint` in order to fix the lint error. Apart from that it looks good to me

File: src/Meteor.js (hunk 1 above)
Comment (bratelefant) [reply src/Meteor.js]: ok, guess I got it.

**Review by TheRealNate — APPROVED** [review]

**Comment by TheRealNate** [comment]
Tentatively approved, but @jankapunkt's changes should be done first

**Review by jankapunkt — COMMENTED** [review]
Code looks good, I will run the network scan today to see if I can confirm the behavior, too. If confirmed I will approve / merge.

**Comment by jankapunkt** [comment]
I will get a new release 2.5.1 out asap

**Pull request merged**